pkg/policy/custompolicybuilder/custom-policy-manager.go

```diff
@@ -0,0 +1,120 @@
+package custompolicybuilder
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type PolicyTemplate struct {
+	Name      string
+	CheckType string
+	Dir       string
+	Type      string
+	Desc      string
+	Title     string
+	Severity  string
+	Category  string
+	RsrcType  string
+}
+
+func (pt *PolicyTemplate) PromptInput() error {
+	var qs = []*survey.Question{
+		{
+			Name: "dir",
+			Prompt: &survey.Input{
+				Message: "Policies directory path",
+				Default: "policies"},
+			Validate: pt.validatePolicyDirectory(),
+		},
+		{
+			Name: "checkType",
+			Prompt: &survey.Select{
+				Message: "Select target:",
+				Options: []string{"terraform", "cloudformation", "kubernetes", "arm"},
+			},
+		},
+		{
+			Name: "name",
+			Prompt: &survey.Input{
+				Message: "policy name",
+				Help:    "Example policy name: my_policy_1"},
+			Validate: pt.validatePolicyName(),
+		},
+		{
+			Name:   "title",
+			Prompt: &survey.Input{Message: "Title"},
```

Comment:
I was thinking isn't better to have some kind of validation for title as well. So it helps in the UI display as well. Most of the tools like checkov and tfsec initially had very large titles like description and later made them short.

Maybe we can have some validation like min length and max length etc.

Reply:
Added validation for mac characters = 57 (57 taken from title validation in assessments.go)